sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -392,6 +393,33 @@ def validate(self, validator):
     return errors
 
 
+class HdfsOptions(PipelineOptions):
```

Comment:
Let's call this HadoopFileSystemOptions to match the name of the file-system.

Reply:
Done.